src/styles/reset.scss

```diff
@@ -0,0 +1,5 @@
+*,
```

Comment:
I'd say it might be in the normalize.scss as well
It's better to create a separate file for fonts (see comment above)